src/commands/distribute/releases/edit.ts

```diff
@@ -0,0 +1,71 @@
+import { AppCommand, CommandResult, ErrorCodes, failure, help, success, shortName, longName, required, hasArg, position, name } from "../../../util/commandline";
+import { AppCenterClient, clientRequest, models } from "../../../util/apis";
+import { out } from "../../../util/interaction";
+import { inspect } from "util";
+
+const debug = require("debug")("appcenter-cli:commands:distribute:releases:delete");
+
+@help("Enables or disables specified release")
+export default class EditReleaseCommand extends AppCommand {
+  @help("Release ID")
+  @shortName("r")
+  @longName("release-id")
+  @required
+  @hasArg
+  public releaseId: string;
+
+  @help("Release state")
+  @name("State")
+  @position(0)
+  @required
+  public state: string;
+
+  public async run(client: AppCenterClient): Promise<CommandResult> {
+    const app = this.app;
+
+    const releaseId = Number(this.releaseId);
+    if (!Number.isSafeInteger(releaseId) || releaseId <= 0) {
+      return failure(ErrorCodes.InvalidParameter, `${this.releaseId} is not a valid release id`);
+    }
+
+    const state = this.state;
+    if (state !== "enabled" && state !== "disabled") {
+      return failure(ErrorCodes.InvalidParameter, `"${state}" is not a valid release state. Available states are "enabled" or "disabled".`);
+    }
+
+    let releaseDetails: models.ReleaseDetailsResponse;
+    try {
+      debug("Loading release details");
+      const httpRequest = await out.progress("Loading release details...", clientRequest<models.ReleaseDetailsResponse>(
+        (cb) => client.releases.getLatestByUser(this.releaseId, app.ownerName, app.appName, cb)
+      ));
+      if (httpRequest.response.statusCode >= 400) {
+        throw httpRequest.response.statusCode;
```

Comment:
I'm not super keen on throwing a number like this - I'd rather handle the error cases here directly and only handle legitimate exceptions in the `catch` block.

Reply:
I see that we do this in other places too - still don't like it but if it's standard then don't worry about this one.

Reply:
Yeah, I don't like it too much, either but I think this pattern was copied from another command which was using it.

Reply:
Yeah, I noticed that after I submitted the review 😆 Let's worry about this another day!

Reply:
we resolved it for this task! fair feedback